network-config-analyzer/CalicoNetworkPolicy.py

```diff
@@ -75,7 +75,8 @@ def __init__(self, name, namespace):
         self.order = None  # None means infinity here
 
     def __eq__(self, other):
-        return super().__eq__(other) and self.order == other.order
+        return super().__eq__(other) and isinstance(other, CalicoNetworkPolicy) and \
+            self.order == other.order
```

Comment:
```suggestion
        return isinstance(other, CalicoNetworkPolicy) and super().__eq__(other) and \
            self.order == other.order
```
Can avoid calling  super().__eq__ if the other policy type does not match.